fdbserver/ApplyMetadataMutation.cpp

```diff
@@ -1070,6 +1079,17 @@ class ApplyMetadataMutationsImpl {
 
 				auto startItr = tenantMap->lower_bound(startTenant);
 				auto endItr = tenantMap->lower_bound(endTenant);
+
+				if (tenantIdIndex) {
+					// Iterate over iterator-range and remove entries from TenantIdName map
+					// TODO: O(n) operation, optimize cpu
```

Comment:
nit: there's probably no way to optimize it?

Reply:
So far I don't think one, but, just want to record it as a TODO for later.